COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -42,10 +42,33 @@ else
   true
 end
 
+def initialize_topics(msg_yaml, env = "production")
+  require "shellwords"
+
+  host, port = msg_yaml[env].values_at("host", "port")
+  host ||= "127.0.0.1"
+  port ||= 9092
```

Comment:
Minor, but I thikn this will read better as the following (same goes for messagin_ready? method)

```
host = msg_yaml[env].fetch("host", "127.0.0.1")
port = msg_yaml[env].fetch("port", 9092)
```